lib/uring/uring_stubs.c

```diff
@@ -524,6 +497,255 @@ ocaml_uring_submit_statx_byte(value* values, int argc) {
 			  values[6]);
 }
 
+// Non-allocating accessors for statx
+// Int64
+int64_t
+ocaml_uring_statx_blksize_native(value v_statx) {
```

Comment:
Might be worth defining a macro to make these functions automatically. e.g. something like:
```c
STATX_GETTER(blksize, int64_t, caml_copy_int64)
STATX_GETTER(attributes, int64_t, caml_copy_int64)
...
```

Reply:
Good idea, did that where feasible here https://github.com/ocaml-multicore/ocaml-uring/pull/97/commits/b12ae5ec19c9a90e4d31cdebc76ac6363f485414